optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/config/BenchmarkProperties.java

```diff
@@ -14,17 +14,35 @@
  * limitations under the License.
  */
 
-package org.optaplanner.spring.boot.autoconfigure;
+package org.optaplanner.spring.boot.autoconfigure.config;
 
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
 
 public class BenchmarkProperties {
+
+    public static final String DEFAULT_SOLVER_BENCHMARK_CONFIG_URL = "solverBenchmarkConfig.xml";
+    public static final String DEFAULT_BENCHMARK_RESULT_DIRECTORY = "target/benchmarks";
+
+    /**
+     * A classpath resource to read the benchmark configuration XML.
+     * Defaults to {@value #DEFAULT_SOLVER_BENCHMARK_CONFIG_URL}.
+     * If this property isn't specified, that {@value #DEFAULT_SOLVER_BENCHMARK_CONFIG_URL} file is optional.
+     */
     private String solverBenchmarkConfigXml;
+
+    /**
+     * The directory to which to write the benchmark HTML report and graphs,
+     * relative from the working directory.
+     */
     private String resultDirectory;
```

Comment:
This is the important part of this PR. The rest is just moving the properties together in a config directory for clarity.